test/Sentry.Tests/Protocol/SentryTransactionTests.cs

```diff
@@ -244,6 +244,30 @@ public void SerializeObject_AllPropertiesSetToNonDefault_SerializesValidObject()
 
             return o;
         });
+
+        Assert.Contains($$"""
```

Comment:
note: I kept this test similar to
https://github.com/getsentry/sentry-dotnet/blob/3745efe4720e76fca6a6f16cddc21f813d2ad911/test/Sentry.Tests/Protocol/SentryEventTests.verify.cs#L6
which also does a serialize/deserialize-roundtrip test
plus an additional "Contains-JSON" check on the serialized string